File: src/main/java/testapp/models/Country.java
package testapp.models;

import com.menighin.luwak.core.interfaces.ILuwakModel;
import lombok.Getter;
import lombok.Setter;

public class Country implements ILuwakModel {

    @Getter @Setter
    private int id;

    @Getter @Setter
    private String name;

    public Country() {

    }

    public Country(int id, String name) {
        this.id = id;
        this.name = name;
    }

}

File: src/main/java/testapp/models/Country.kt
package testapp.models

import com.menighin.luwak.core.interfaces.ILuwakModel

class Country(var id: Long, var code: String?) : ILuwakModel {

    companion object {
        const val ID: String = "id"
        const val CODE: String = "code"
    }

}